release-notes.txt

```diff
@@ -1,3 +1,14 @@
+Versjon 2.0.12, October 2016
+
+* Made ecl_file_view 1st class citizen, exposed to API
+* Added EXCLUDE_HOSTS to LSF driver with dynamic blacklisting
+* Fixed MAX_RUNTIME is respected by WPRO
+* Added functionality to open restart files in RW mode with seek
+* Improved date handling (all is UTC now)
```

Comment:
As a result of this the filesystem version has been upgraded to 107, to use an old case you must upgrade it:  "ert_upgradefs_107  config_file".